frame/dvm/src/lib.rs

```diff
@@ -620,12 +621,11 @@ impl<T: Config> InternalTransactHandler for Pallet<T> {
 		match info {
 			CallOrCreateInfo::Call(info) => match info.exit_reason {
 				ExitReason::Succeed(_) => Ok(info.value),
-				_ => {
-					log::error!("Executing internal transaction error happened");
-					Err(Error::<T>::FailedInternalTx.into())
-				}
+				ExitReason::Error(_) => Err(<Error<T>>::InternalTransactionExitError.into()),
+				ExitReason::Revert(_) => Err(<Error<T>>::InternalTransactionRevertError.into()),
```

Comment:
does this only show the error type `revert`? Can we catch the detail of the error?